mutation_fragment.hh

```diff
@@ -39,6 +39,8 @@ public:
             : _ck(std::move(ck)), _row(std::move(t), std::move(marker), std::move(cells)) {
         _row.maybe_shadow();
     }
+    clustering_row(clustering_key_prefix ck, deletable_row row)
```

Comment:
This could be `clustering_row(const clustering_key&, deletable_row&& row)` to reduce the number of std::move's